Neuro_Annotations.json

```diff
@@ -89,6 +89,9 @@
     "sampleName": [
         
     ],
+    "individual": [
```

Comment:
did we decide to stay with `individual`, or use `individualId` in combination with `individualIdSource`?